app/src/main/java/woowacourse/payments/ui/addcard/PaymentCard.kt

```diff
@@ -2,38 +2,55 @@ package woowacourse.payments.ui.addcard
 
 import androidx.compose.foundation.background
 import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
 import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.layout.size
 import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.draw.shadow
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import woowacourse.payments.domain.BankType
 
 @Composable
-fun PaymentCard(modifier: Modifier = Modifier) {
+fun PaymentCard(
+    bank: BankType,
+    modifier: Modifier = Modifier,
+) {
     Box(
-        contentAlignment = Alignment.CenterStart,
+        contentAlignment = Alignment.TopStart,
         modifier =
             modifier
                 .shadow(8.dp)
                 .size(width = 208.dp, height = 124.dp)
                 .background(
-                    color = Color(0xFF333333),
+                    color = Color(bank.color),
                     shape = RoundedCornerShape(5.dp),
                 ),
     ) {
-        Box(
-            modifier =
-                Modifier
-                    .padding(start = 14.dp, bottom = 10.dp)
-                    .size(width = 40.dp, height = 26.dp)
-                    .background(
-                        color = Color(0xFFCBBA64),
-                        shape = RoundedCornerShape(4.dp),
-                    ),
-        )
+        Column {
+            Text(
+                modifier =
+                    Modifier
+                        .padding(start = 14.dp, bottom = 10.dp, top = 10.dp),
```

Comment:
```suggestion
                    Modifier
                        .padding(start = 14.dp)
                        .padding(vertical = 10.dp),
```

(좌,우) 또는 (상,하) 동일하게 패딩을 줄 때는 horizontal, vertical 로도 패딩을 줄 수 있더라구요 ! 
현재 같은 경우 padding을 2개에 걸쳐서 설정하는 느낌이라서 굳이 일 수 있겠지만 이런 방법도 있다는 걸 알려주고 싶어서 남겼습니다 !

Reply:
저는 padding을 두번 호출하는 것이 좀 보기 불편하다..? 라고 생각했어요! 1번 패딩을 보고 2번 패딩을 봐야 전체적인 패딩을 이해할 수 있어서요!
근데 또 생각해보면 bottom , top 을 따로 보는것보다 vertical으로 하나로 보는게 더 낫지 않나..? 하는 생각도 드네요...

[d7c6cae](https://github.com/woowacourse/android-payments/pull/80/commits/d7c6cae8fe483836106054ef864983e052cd4e1e)

Reply:
이 리뷰는 반영하길 바랬다기보다는 이런 정보도 있다는 걸 알려주고 싶었어요 !